src/components/screens/SUE/SUEDetailScreen.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export const SUEDetailScreen = ({ data, navigation, route }) => {
+  return (
+    <View>
+      <Text>DETAIL</Text>
+    </View>
+  );
```

Comment:
TODO?

Reply:
Thank you for the reminder. I don't see anything important for TODO. These screens will be developed and ready in a few days.